### PR TITLE
 Support changing cacheVolume of cn and dn after mo is deployed

### PR DESCRIPTION
**Pull request by yangj1211**
## What type of PR is this?

- [ ] Enhancement
- [ ] Displaying
- [ ] Typo
- [x] Doc Request

## Which issue(s) this PR fixes:

issue #1167

### CODE REVIEW

**Comment by yangj1211** [comment]
@aronchanisme need to review

File: docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-with-k8.md [hunk 1]
```diff
@@ -499,6 +499,45 @@ NS="mo"
...
 
+### 动态扩容
+
+Operator 支持 TN、CN 的 cacheVolume 配置动态扩容，不支持缩容操作，扩容过程前后，CN、TN pod 无需重启。具体步骤如下：
+
+1. 确保 StorageClass 支持卷扩展功能
+
+    ```bash
+    #获取 storageclass 信息
+    >kubectl get storageclass ${SC_NAME} -oyaml #SC_NAME 是 mo 集群使用的 pvc 的 sc 类型
```
Comment (aronchanisme) [inline docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-with-k8.md:510]: 可以提示一下，用这个命令可以查看`STORAGE_CLASS`的名称
[root@VM-0-122-tencentos ~]# kubectl get pvc -n ${MO_NS}
NAME                                       STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
mo-data-mo-customer-test-cnset1-cn-rhv55   Bound    pvc-5b8ed11a-bd16-4003-9d2e-0b17de76e4b9   100Gi      RWO            cbs-hssd       3h10m
mo-data-mo-customer-test-cnset1-cn-sddmf   Bound    pvc-804f64c4-c167-45b4-b82c-7af966a36971   100Gi      RWO            cbs-hssd       3h10m
mo-data-mo-customer-test-dn-0              Bound    pvc-86d35491-b18c-4ad6-a685-95da970d211c   50Gi       RWO            cbs-hssd       3h10m
mo-data-mo-customer-test-log-0             Bound    pvc-b6888a78-c946-498c-935e-cd1ae333c11a   100Gi      RWO            cbs-hssd       3h11m
mo-data-mo-customer-test-log-1             Bound    pvc-fad96b55-b55e-4768-a80b-1d8e6168a2db   100Gi      RWO            cbs-hssd       3h11m
mo-data-mo-customer-test-log-2             Bound    pvc-12416574-e763-4124-bebf-033e69bd2838   100Gi      RWO            cbs-hssd       3h11m

File: docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-with-k8.md [hunk 2]
```diff
@@ -499,6 +499,45 @@ NS="mo"
...
+    ```
+
+2. 进入集群配置编辑模式：
+
+    ```bash
+    kubectl edit mo -n ${MO_NS} ${MO_NAME} # 其中 MO_NS 为部署 MO 集群的命名空间，MO_NAME 为 MO 集群的名称；例如 MO_NS=matrixone; MO_NAME=mo_cluster
+    ```
+
+3. 按需修改 tn 和 cn 的 cacheVolume 的 size 大小。
+
```
Comment (aronchanisme) [inline docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-with-k8.md:524]: 视乎修改的对象：
1、如果为CN group，则修改`spec.cnGroups[0].cacheVolume`（或`spec.cnGroups[1].cacheVolume`，其中`[n]`中的`n`为CN Group的数组下标）；
2、如果为CN，则修改`spec.tp.cacheVolume`
3、如果为TN，则修改`spec.tn.cacheVolume`（或`spec.dn.cacheVolume`）

File: docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-without-k8.md [hunk 3]
```diff
@@ -688,6 +688,45 @@ __Note:__ 本章节均是在 master0 节点操作。
...
+
+    然后保存并退出：按 esq 键，和：wq
+
+4. 查看扩容结果
+
+    ```bash
+    
+    >kubectl get pvc -n ${MO_NS}
+
+    >kubectl get pv
```
Comment (aronchanisme) [inline docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-without-k8.md:727]: kubectl get pv | grep ${NS}

File: docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-without-k8.md [hunk 4]
```diff
@@ -688,6 +688,45 @@ __Note:__ 本章节均是在 master0 节点操作。
...
+3. 按需修改 tn 和 cn 的 cacheVolume 的 size 大小。
+
+    ```bash
+    - cacheVolume:
+            size: 900Gi
+    ```
+
+    然后保存并退出：按 esq 键，和：wq
+
+4. 查看扩容结果
```
Comment (aronchanisme) [inline docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-without-k8.md:721]: 提示注意`CAPACITY`这个输出

File: docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-with-k8.md [hunk 5]
```diff
@@ -499,6 +499,45 @@ NS="mo"
...
+    ```bash
+    - cacheVolume:
+            size: 900Gi
+    ```
+
+    然后保存并退出：按 esq 键，和：wq
```
Comment (aronchanisme) [inline docs/MatrixOne/Deploy/deploy-Matrixone-cluster/deploy-MatrixOne-cluster-with-k8.md:530]: 是英文的`:`

**Review by aronchanisme — COMMENTED** [review]
再看下？

**Comment by yangj1211** [comment]
@aronchanisme fixed，pls check again

**Review by aronchanisme — APPROVED** [review]
ok

**Pull request merged**